changelog.md

```diff
@@ -8,6 +8,13 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.0.15 September 2, 2019
+* Added a dark content_css styles to go with the dark skin.
+* Changed so the enabled state on toolbar buttons doesn't get the hover effect.
```

Comment:
Changed the enabled state on toolbar buttons so they don't get the hover effect.  ?